### PR TITLE
Increase ServiceHub log verbosity for integration tests

### PR DESCRIPTION
**Pull request by sharwell**

### CODE REVIEW

File: eng/build.ps1 [hunk 1]
```diff
@@ -573,6 +573,7 @@ function Setup-IntegrationTestRun() {
     Capture-Screenshot $screenshotPath
   }
 
+  $env:ServiceHubTraceLevel = "Information"
```
Comment (Cosifne) [inline eng/build.ps1:576]: How could the log be viewed? Do we need to log to the cloud run machine to see it?

File: azure-pipelines-integration.yml [hunk 2]
```diff
@@ -59,7 +59,7 @@ jobs:
         ArtifactName: '$(System.JobAttempt)-Logs $(_configuration) OOP64_$(_oop64bit) $(Build.BuildNumber)'
         publishLocation: Container
       continueOnError: true
-      condition: not(succeeded())
```
Comment (Cosifne) [inline azure-pipelines-integration.yml:62]: How can the servicehub log be published? Or we need some other way to view it?

File: eng/build.ps1 (hunk 1 above)
Comment (Cosifne) [reply eng/build.ps1:576]: Diagnostics level might generate a lot of log so it might be harmful. But just curious how dangerous it is?

**Review by Cosifne — APPROVED** [review]
I just have some questions about the details of the change

File: azure-pipelines-integration.yml (hunk 2 above)
Comment (sharwell) [reply azure-pipelines-integration.yml:62]: Right, it gets attached as a build artifact

**Review by JoeRobich — APPROVED** [review]

**Comment by sharwell** [comment]
Marked as draft since the amount of data collect was difficult to use

**Pull request closed without merging**